jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -760,7 +780,7 @@ ecma_op_general_object_define_own_property (ecma_object_t *obj_p, /**< the objec
                                                        NULL,
                                                        NULL,
                                                        was_enumerable,
-                                                       was_configurable);
+                                                       is_current_configurable);
```

Comment:
This is tricky here. If is_current_data_descriptor is true, we deleted a data descriptor, and create an accessor. We have no other choice.